.github/workflows/schemas.yml

```diff
@@ -60,41 +59,12 @@ jobs:
   pipeline-yaml:
     runs-on: ubuntu-latest
     steps:
-
-      - uses: actions/checkout@v2
-
-      - name: Setup Python
-        uses: ./.github/actions/python_cache/
-
-      - name: Install sndfile
-        run: sudo apt update && sudo apt-get install libsndfile1 ffmpeg
-
-      - name: Install Haystack
-        run: |
-          pip install --upgrade pip
-          pip install -U .[all]
-
-      - name: Update pipeline YAML schemas
-        run: python .github/utils/generate_json_schema.py
-
-      - name: Check status
-        run: |
-          if [[ `git status --porcelain` ]]; then
-            git status
-            echo "##################################################################################################"
-            git diff
-            echo "##################################################################################################"
-            echo "# "
-            echo "# CHECK FAILED! The YAML schemas for pipelines were not updated."
-            echo "# "
-            echo "# Please generate the new schemas locally:"
-            echo "# "
-            echo "#     python .github/utils/generate_json_schema.py"
-            echo "# "
-            echo "# Or see https://github.com/deepset-ai/haystack/blob/main/CONTRIBUTING.md for help."
-            echo "# "
-            echo "# If you have further problems, please open an issue: https://github.com/deepset-ai/haystack/issues"
-            echo "# "
-            echo "##################################################################################################"
-            exit 1
-          fi
+      # Start a workflow in https://github.com/deepset-ai/haystack-json-schema 
+      # https://docs.github.com/en/rest/repos/repos#create-a-repository-dispatch-event
+      - run: |
+          curl \
+          -X POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "Authorization: Bearer ${{ env.ACCESS_TOKEN }}" \
```

Comment:
let's use directly a secret, can you try `HAYSTACK_BOT_TOKEN`?

Reply:
Works! 
![image](https://user-images.githubusercontent.com/5703634/198340809-03d5caef-e0a1-4f8a-aacd-a23b9f2d0a8a.png)